var/spack/repos/builtin/packages/scine-sparrow/package.py

```diff
@@ -8,7 +8,7 @@
 from spack.package import *
 
 
-class Sparrow(CMakePackage):
+class ScineSparrow(CMakePackage):
```

Comment:
@frobnitzem What are your thoughts on the name change?

Reply:
This makes good sense, since it reflects the group of packages sparrow is a part of.   @awvwgk has done the hard work of putting all the dependencies into spack recipes, but it will take some time to review them.

Reply:
We may want to keep the old package and deprecate all versions, in addition to the new package. This will prevent surprises for users expecting the old name. See https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions

Reply:
Done, there is now a (deprecated) `sparrow@3.0.0` package which depends on `scine-sparrow+python@3.0.0`.